recipes/libvips/meta.yaml

```diff
@@ -0,0 +1,103 @@
+# Notes
+# 
+# - `niftiio`, `openexr` and `libheif` are missing
+# 
+# - `imagemagick` is built with `jpeg`, not `libjpeg-turbo`, so we must use
+#   `jpeg` too. This is very unfortunate. It would be good if IM could move
+#   to turbo as well.
+
+{% set name = "libvips" %}
+{% set version = "8.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libvips/libvips/releases/download/v{{ version }}/vips-{{ version }}.tar.gz
+  sha256: a0ee255a2a1ebfea5b2dff2a780824d7157a78c010d7ddd531279aacefbf2539 
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    # SO name changes at minor rev bumps.
+    - {{ pin_subpackage('libvips', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config        # [not win]
+  host:
+    - bzip2 
+    - cairo 
+    - cfitsio 
+    - expat
+    - fftw
+    - fontconfig
+    - freetype
+    - gdk-pixbuf
+    - giflib
+    - glib
+    - gstreamer-orc
+    - hdf5
+    - imagemagick
+    - jpeg
+    - lcms2
+    - libexif
+    - libgsf
+    - libmatio
+    - libpng
+    - librsvg 
+    - libtiff
+    - libwebp
+    - openslide
+    - pango
+    - poppler
+    - zlib
+  run:
+    - fftw
+    - gdk-pixbuf
+    - gstreamer-orc
+    - libexif
+    - libgsf
+    - libmatio
+    - librsvg 
+    - libwebp
+    - libxml2
+    - imagemagick
+    - openslide
+    - pango
+    - poppler
+
+test:
+  commands:
+    - vips --version
+
+about:
+  home: https://libvips.github.io/libvips
+  license: LGPL-2.1
+  license_family: LGPL
+  license_file: COPYING
+  summary: 'A fast image processing library with low memory needs'
+  description: |
+    libvips is a demand-driven, horizontally threaded image processing
+    library. Compared to similar libraries, libvips runs quickly and uses
+    little memory.  It has around 300 operations covering arithmetic,
+    histograms, convolution, morphological operations, frequency filtering,
+    colour, resampling, statistics and others. It supports a large range of
+    numeric formats, from 8-bit int to 128-bit complex. Images can have any
+    number of bands. It supports a good range of image formats, including
+    JPEG, TIFF, OME-TIFF, PNG, WebP, FITS, Matlab, PDF, SVG, HDR,
+    PPM, CSV, GIF, Analyze, DeepZoom, and OpenSlide. It can also load images
+    via ImageMagick or GraphicsMagick, letting it load formats like DICOM.
+  doc_url: https://libvips.github.io/libvips/API/current
+  dev_url: https://github.com/libvips/libvips
+
+extra:
+  recipe-maintainers:
+    - sebastian-luna-valero
```

Comment:
@sebastian-luna-valero @strets123 Are you happy to be listed as maintainers of this recipe?

Reply:
They volunteered in this thread: https://github.com/libvips/libvips/issues/673 though it wouldn't hurt to ask again.

Reply:
Yes, happy to keep involved. Thanks everyone!

Reply:
Looking at stret123's profile page, he's been quiet for two weeks, I guess he's on holiday. I'll remove his name for now, and perhaps replace if it he asks. There will be a 8.8.2 rather soon.

Reply:
That's good enough for me, I had only checked the participants on this page.